devel/lpc21isp/patches/120-nanosleep.patch

```diff
@@ -0,0 +1,12 @@
+--- a/lpc21isp.c
++++ b/lpc21isp.c
+@@ -1130,7 +1130,8 @@ void ClearSerialPortBuffers(ISP_ENVIRONMENT *IspEnvironment)
+ */
+ void Sleep(unsigned long MilliSeconds)
+ {
+-    usleep(MilliSeconds*1000); //convert to microseconds
++    struct timespec m = {MilliSeconds * 1000 * 1000, 0};
```

Comment:
I thought the format here was `{seconds, nanoseconds}`?

Reply:
Fixed.